source/development_process/1_development.rst

```diff
@@ -0,0 +1,466 @@
+1. How a Project is Written
+===========================
+This is a guide for writing a Django website. It will not cover
+how to write a website using Flask because the process is nearly identical and
+the team has indicated that Django is to be preferred for new projects.
```

Comment:
This isn't really true, we will use Flask, or whatever tool seems to fit best. I think Django is fine as an example app to walk through the process, but lets not make a statement about what tool is preferred